hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/steps/EstimateFeature.java

```diff
@@ -385,43 +406,93 @@ public void limitedNestedCallsFunction() {
                 new BigInteger("1"),
                 new BigInteger("500"),
                 asAddress(contractSolidityAddress));
-        validateGasEstimation(data, NESTED_CALLS_LIMITED, contractSolidityAddress);
+        validateGasEstimationNestedCalls(data, NESTED_CALLS_LIMITED, contractSolidityAddress);
         var secondData = encodeData(
                 ESTIMATE_GAS,
                 NESTED_CALLS_LIMITED,
                 new BigInteger("1"),
                 new BigInteger("1024"),
                 asAddress(contractSolidityAddress));
-        validateGasEstimation(secondData, NESTED_CALLS_LIMITED, contractSolidityAddress);
+        validateGasEstimationNestedCalls(secondData, NESTED_CALLS_LIMITED, contractSolidityAddress);
         var thirdData = encodeData(
                 ESTIMATE_GAS,
                 NESTED_CALLS_LIMITED,
                 new BigInteger("1"),
                 new BigInteger("1025"),
                 asAddress(contractSolidityAddress));
-        validateGasEstimation(thirdData, NESTED_CALLS_LIMITED, contractSolidityAddress);
+        validateGasEstimationNestedCalls(thirdData, NESTED_CALLS_LIMITED, contractSolidityAddress);
+    }
+
+    @And("I approve the contract to use FT")
+    public void approveFungibleWithERCContract() {
+        final var ercTestContractId = AccountId.fromSolidityAddress(ercSolidityAddress);
```

Comment:
Right now the sdk method doesn't work properly with non-zero realm.

```suggestion
        final var ercTestContractId = TestUtil.fromSolidityAddress(ercSolidityAddress);
```

Reply:
Done